test/HeadersTest.php

```diff
@@ -381,6 +381,7 @@ public function testHeaderCrLfAttackAddHeaderLineWithValue()
         $headers = new Mail\Headers();
         $this->setExpectedException('Zend\Mail\Header\Exception\InvalidArgumentException');
         $headers->addHeaderLine('Fake', "foo-bar\r\n\r\nevilContent");
+        $headers->forceLoading();
```

Comment:
So include that information in the pull request summary, as that looks like the important change (or one of the important changes) you're introducing.